cleaning_data.py

```diff
@@ -0,0 +1,199 @@
+"""change documents to UTF-8"""
+
+
+"""tohle má ostatní texty uložit do UTF-8 formátu
+Taky by to mělo vymazat všechno, kromě textu knihy ze zdrojů gutenberg"""

```

Comment:
please do not mix languages and merge them to simple lines
feel free to use chatGPT to do texting